bin/start_be.sh

```diff
@@ -55,14 +55,14 @@ rm -f ${UDF_RUNTIME_DIR}/*
 pidfile=$PID_DIR/be.pid
 
 if [ -f $pidfile ]; then
-    if kill -0 $(cat $pidfile); then
+    if [ -d /proc/$(cat $pidfile) ]; then
```

Comment:
proc filesystem maybe not mount in some environment. 
can you change it to 
```suggestion
    if kill -0 $(cat $pidfile) > /dev/null 2>&1; then
```